llvm/tools/llvm-exegesis/lib/CMakeLists.txt

```diff
@@ -12,6 +12,9 @@ endif()
 if (LLVM_TARGETS_TO_BUILD MATCHES "Mips")
   list(APPEND LLVM_EXEGESIS_TARGETS "Mips")
 endif()
+if(LLVM_TARGETS_TO_BUILD MATCHES "RISCV" OR LLVM_EXPERIMENTAL_TARGETS_TO_BUILD MATCHES "RISCV")
```

Comment:
Why would RISCV be in LLVM_EXPERIMENTAL_TARGETS_TO_BUILD? It's not an experimental target.

Reply:
Addressed